tests/unit/test_routes/test_categories.py

```diff
@@ -0,0 +1,63 @@
+from app.utils import get_error_code_from_status
+
+
+def test_categories(module_client, module_db):
+    client = module_client
+
+    response = client.get('api/v1/categories')
+
+    for category in response.json['data']:
+        assert (isinstance(category.get('id'), int))
+        assert (isinstance(category.get('name'), str))
+        assert (category.get('name'))
+    assert (response.json['number_of_pages'] is not None)
+
+    # Test trying to get a page of results that doesn't exist
+    too_far = 99999999
```

Comment:
We should break out this negative test into its own function

Reply:
The actual paginator is tested in test_utils. I would suggest just deleting the second half of this test since it adds no value.

Reply:
It probably doesn't add a lot of value because of the paginator tests in test_utils, but I would prefer to keep it just as a warm and fuzzy that every endpoint implements pagination correctly, so I'd like to keep it